src/asf_tools/water_map.py

```diff
@@ -390,6 +390,15 @@ def hyp3():
 
     log.info(f'Water map created successfully: {water_map_raster}')
 
+    files_to_remove = [
+        Path(product_name) / f'{product_name}_WM_VV_initial.tif',
+        Path(product_name) / f'{product_name}_WM_VH_initial.tif',
+        Path(product_name) / f'{product_name}_WM_VV_fuzzy.tif',
+        Path(product_name) / f'{product_name}_WM_VH_fuzzy.tif',
```

Comment:
`product_name` already includes `_WM` (see L378 above)
```suggestion
        Path(product_name) / f'{product_name}_VV_initial.tif',
        Path(product_name) / f'{product_name}_VH_initial.tif',
        Path(product_name) / f'{product_name}_VV_fuzzy.tif',
        Path(product_name) / f'{product_name}_VH_fuzzy.tif',
```